test/models/playlist_test.rb

```diff
@@ -0,0 +1,74 @@
+require 'test_helper'
+
+class PlaylistTest < ActiveSupport::TestCase
+  def valid_playlist_params
+    {
+      name: 'name1',
+      energy: 35,
+      danceability: 56,
+      genre: 'genre1'
+    }
+  end
+
+  test 'valid with a name' do
+    playlist = Playlist.new(valid_playlist_params.merge(name: 'name1'))
+    playlist.valid?
+  end
+
+  test 'invalid without name' do
+    playlist = Playlist.new(valid_playlist_params.merge(name: 'name1'))
+    assert_not_nil playlist.errors[:name]
+  end
+
+  test 'valid with all parameters' do
+    playlist = Playlist.new(valid_playlist_params.merge(name: 'name1', genre: 'genre1', danceability: 23, energy: 12))
+    playlist.valid?
```

Comment:
sans `assert` dans le test ça ne test rien :)

ce commentaire est valide pour les 3 tests du dessous

```suggestion
    assert playlist.valid?
```